terraform/aws/main.tf

```
@@ -103,7 +103,8 @@ resource "aws_instance" "peer" {
   }
 
   tags = {
-    Name = local.name
+    Name  = local.name
+    owner = system
```

```suggestion
    owner = "system"
```

I *think* this needs to be quoted to be interpreted as a string.